Description: The manpage shows methods that don't work. They have been replaced.
From: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=447402
Forwarded: no
--- a/vorbisgain.1
+++ b/vorbisgain.1
@@ -110,7 +110,7 @@
 shell to not do any filename globbing:
 .PP
 .RS
-vorbisgain \-a \-f \-r "music/*.ogg"
+vorbisgain \-a \-f \-r music/
 .RE
 .PP
 Calculate the album gain. The files specified before the directory "album"
@@ -124,7 +124,7 @@
 Remove all replaygain tags from a collection of oggs:
 .PP
 .RS
-vorbisgain \-c \-r "music/*.ogg"
+vorbisgain \-c \-r music/
 .RE
 .SH TAG FORMAT
 .B vorbisgain
